### ISSUE TITLE
Typo in Make file + No UEFI boot

### ISSUE THREAD

**Issue by bert003** [redorescue/redorescue#133]
Line **541** has to change from` if [ "$NONFREE" = true ]; then` to` if [ "$NONFREE" == true ]; then`

Also, UEFI boot does not work. I have all the requirements installed and the ISO compiles successfully. I compared version 4 and version 5 and the only differences seem to be in the grub EFI files. They are much larger in size.

Any feedback?

**Comment by nros** [comment]
I do not think that this change is a "must do". Actually, the Bash syntax allows single "=" as well, because POSIX standard uses single "=".

see:
* Bash documentation https://www.gnu.org/software/bash/manual/bash.html#Bash-Conditional-Expressions
* POSIX standard: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html
* https://www.baeldung.com/linux/conditional-expressions-shell-script
*
